content/blog/2021/08/using-maxsat-for-resolving-deps.md

```diff
@@ -0,0 +1,165 @@
+# Using a MaxSAT for resolving Hypper chart dependencies on Kubernetes
+
+[Hypper](http://hypper.io) is a Kubernetes package manager based on Helm. It
+handles charts in a similar way as Helm does, with a twist: it tries to resolve
+dependencies between charts to reuse existing resources and deployments.
+For more info, see the two articles on this blog by @mattfarina: [Introducing
+Hypper](https://community.suse.com/posts/15893197) and
+[Hypper v3 - Now with a SAT solver](https://community.suse.com/posts/14072167).
+
+Hypper is designed to work with shared dependencies: dependencies installed once
+on a cluster that several charts depend on. Think of Hypper as bringing the
+usual Linux package management to Kubernetes Helm charts.
+
+Checking all the charts so see if their dependencies are correctly installed or
+there's some missing is not an easy task. This is called a "Satisfiability
```

Comment:
there are (it's multiple dependencies)

Reply:
Also if we publish on hypper.io first we need to update the references to the posts from @mattfarina to the suse community blog.

Reply:
Couldn't we keep the existing references to the existing posts from Matt as they keep being there? I would not like to migrate all past posts to a hypper.io/blog.

Reply:
That was not my point, I just meant we should say explicitly in the references to the previous post, that those are hosted on the community blog (to avoid confusion for the reader).